packages/common/src/dto/availability-information.dto.ts

```diff
@@ -26,9 +33,17 @@ export class AvailabilityDTO {
   })
   deploymentLocations!: LhaId[];
 
-  placementPrefs!: PlacementPreferencesDTO;
+  @IsArray({ message: 'Placement options are required' })
+  @ArrayMinSize(1, { message: 'Placement options are required' })
+  @ArrayMaxSize(Object.values(PlacementOptions).length, {
+    message: 'Invalid placement options',
+  })
```

Comment:
Does this currently validate the contents of the array?

Reply:
No I don't think so! Good catch